detectors/node/opentelemetry-resource-detector-aws/README.md

```diff
@@ -37,6 +37,20 @@ const tracerProvider = new NodeTracerProvider({ resource });
 - `awsLambdaDetector`: Populates `faas` and `cloud` for functions running on [AWS Lambda](https://aws.amazon.com/lambda/)
   - `faas.id` is currently not populated as it is not provided by the runtime at startup
 
+## Ordering
+
+Note that if multiple detectors are inserting the same attribute name, the first detector to insert wins.
+For example, if you had `detectors: [AwsEksDetector, AwsEc2Detector]` then `cloud.platform` will be `eks` instead of `ec2`.
+The below ordering is recommended.
+
+### AWS Detectors
```

Comment:
I think this header is redundant, as all the detectors in this package are AWS.
The [reference](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/resourcedetectionprocessor/README.md#ordering) in collector contrib lists AWS and GCP, but here we can just enumerate them directly.

Reply:
fixed

Reply:
Can you please push it? :)